kmath-geometry/src/commonMain/kotlin/scientifik/kmath/geometry/VectorSpace.kt

```diff
@@ -0,0 +1,47 @@
+package scientifik.kmath.geometry
+
+import scientifik.kmath.dimensions.Dimension
+import scientifik.kmath.linear.Point
+import scientifik.kmath.operations.ExtendedField
+import scientifik.kmath.operations.Space
+
+interface Vector<T, D : Dimension> : Point<T>
+
+interface VectorSpace<T, D : Dimension, V : Vector<T, D>> : Space<V> {
```

Comment:
Probably should be reconciled with the `VectorSpace` interface from `linear` package